src/librustdoc/passes/collect_intra_doc_links.rs

```diff
@@ -564,6 +564,7 @@ impl<'tcx> LinkCollector<'_, 'tcx> {
             ty::Adt(ty::AdtDef(Interned(&ty::AdtDefData { did, .. }, _)), _) | ty::Foreign(did) => {
                 Res::from_def_id(self.cx.tcx, did)
             }
+            ty::Field(..) => bug!("FIXME(field_projections): what to do here?"),
```

Comment:
Not calling for action right away: I feel like we can just return `None` for now.

Reply:
Similarly here, but I didn't really test rustdoc at all...